### ISSUE TITLE
Doesn't trigger changing without opening a file

### ISSUE THREAD

**Issue by mlorthiois** [ahmedkhalf/project.nvim#5]
Hello,

First, thanks for your plugin. 

As said on reddit, project.nvim doesn't provide the same functionality as vim-rooter. I have the 0.5.0 stable version. If my project is like this : 
```
.
├── .git
├── install.sh
├── nvim 
│   ├── init.vim
│   ├── lua <--------------------------- nvim lauched from here
│   │   ├── completion.lua
│   │   ├── telescope-config.lua
│   │   ├── treesitter.lua
│   │   └── whichkey.lua
```

If I open NeoVim (with `nvim` or `nvim .`) from the subdirectory `nvim/lua`, at opening, vim-rooter change my cwd, and `:Telescope find_files` can fuzzy find `install.sh` and `init.vim`. So I don't have to worry about where I open neovim.

That's not the case with project.nvim, where I have to for example open a file, for example `completion.lua`, to change the cwd. Without this opening, my cwd stays at `nvim/lua` and `:Telescope find_files` can't find `install.sh` and `init.vim`.

Hope that makes sense. Thanks!

**Comment by beauwilliams** [comment]
@mlorthiois I can add some filetypes into my last pr if your would like. Are you launching nvim to a dashboard? That's what I usually do with startify then launch telescope and fuzzy find. 
With my pr right now launching nvim with startify installed will activate the plugin. No need to open a file.

**Comment by mlorthiois** [comment]
Not really, I get to that startup screen and use my telescope mappings for navigation.
<img width="1438" alt="Capture d’écran 2021-08-14 à 16 29 27" src="https://user-images.githubusercontent.com/48413507/129449612-dd3dcba8-866d-44f0-a182-c8dade69c28f.png">
and I don't know if a filetype is linked to this startup screen... 

But I think you can add the filetypes `netrw` when neovim is launched with `nvim .` and `dashboard` if dashboard-nvim is used.

**Comment by n3wborn** [comment]
Sorry, not sure what really works for me if it works.
When from dir_A I type `nvim dir_B`  I can see that nvim-tree shows dir_B but Telescope find_file  still shows files from dir_A.
When, from dir_A I type `nvim_B/file.lua` I'm just editing the file while still being in dir_A. I've kept default settings as shown in README, with  NVIM v0.6.0-dev+135-g222cd4322

**Comment by beauwilliams** [comment]
I think perhaps removing the is_file check might be the least complex solution here. I'm not sure why the developer created that check, but it would be easiest if we just remove it. Then the cwd should update regardless of how you launch vim and if a file was opened upon launch before opening up say telescope.

**Comment by beauwilliams** [comment]
Oh well just created another simple pr. Up to dev what they think is best. I think we can do without a file check. But again, not sure why dev included it in first place.

**Comment by ahmedkhalf** [comment]
![image](https://user-images.githubusercontent.com/36672196/129486070-52793f47-8378-4cef-bf10-da2213b98692.png)

The reason why I added the is_file check is mainly a micro-optimization so I don't waste CPU cycles for useless buffers like this one shown above. Imagine someone holding the tab key on this, and the number of calculations that would make when many buffers are created and destroyed.

Sry @beauwilliams but I am gonna have to close the PR's you submitted, instead I decided to just relax the restrictions of is_file instead of flat out removing it.

@n3wborn let me know if this issue is fixed!
